packages/test/test-service-load/package.json

```diff
@@ -38,7 +38,7 @@
 		"start": "node ./dist/nodeStressTest.js",
 		"start:frs": "node ./dist/nodeStressTest.js --driver routerlicious --driverEndpoint frs --profile ci_frs",
 		"start:mini": "node ./dist/nodeStressTest.js  --profile mini",
-		"start:odsp": "node ./dist/nodeStressTest.js --driver odsp",
+		"start:odsp": "node ./dist/nodeStressTest.js --driver odsp --driverEndpoint odsp",
```

Comment:
@NicholasCouri , can you please help me understand how it works? When I search for "driverEndpoint" in our repo, I see things like type DriverEndpoint = RouterliciousEndpoint | OdspEndpoint.
I do see places where we use odsp-df strings in code, but it feels like it's a bit messed up.

I feel like we driverEndpoint should be either frs or odsp.
But we also need to add "ring" sub-item, that can be odsp or odsp-df (and OdspEndpoint should be OdspRing, not end-point).

What do you think? Should we open an item to do so? It's not just about using consistent names, it also makes it much easier to work with telemetry, as today we search for "driver" when searching for odsp data, and for "driverEndpoint" when searching for afr data - very inconsistent and confusing.